profile_library/signify/LTV004/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-07-18T03:17:17.110351",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plus Plug US (SNPL-00116US)",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 10,
+    "SLEEP_TIME": 5,
+    "VERSION": "v1.17.14:docker"
+  },
+  "name": "Hue white filament Edison ST19 E26 LED warm-to-cool",
+  "standby_power": 0.18
```

Comment:
```suggestion
  "standby_power": 0.18,
  "aliases": [
    "929002478401",
    "046677563585"
  ]
```

Reply:
Let's add the aliases as seen on https://zigbee.blakadder.com/Philips_LTV004.html
Zigbee2Mqtt exposes these as model numbers, so this way we also make sure auto discovery will work in this case.